pkg/framework/didresolver/resolver.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package resolver
+
+import (
+	"strings"
+
+	diddoc "github.com/hyperledger/aries-framework-go/pkg/doc/did"
+	"github.com/pkg/errors"
+)
+
+// Resolver did resolver
+type Resolver struct {
```

Comment:
@troyronda @fqutishat this stutters: `didresolver.Resolver`

Reply:
@troyronda @fqutishat this should ideally be typed as `did.Resolver`

Reply:
should I change didresolver to did

Reply:
It can be okay for the main struct of a dedicated package to stutter with the package itself.
(But in that case, I would have expected an exact match like didresolver.DIDResolver).

Reply:
(Note that the usage is actually didresolver.New in this case).

Reply:
change to DIDResolver